torch/csrc/autograd/VariableTypeManual.cpp

```diff
@@ -213,14 +212,19 @@ Tensor & copy_(Tensor & self, const Tensor & src, bool non_blocking) {
   }
   {
     at::AutoNonVariableTypeMode non_var_type_mode(true);
```

Comment:
I think we don't need this anymore under the new scheme (same for others here in `VariableTypeManual.cpp`, but also in codegen)

Reply:
Oh, cool- what replaces that logic (or can I just remove it completely)? Doesn't there need to be something that adds autograd keys to the exclude set, so we won't re-use them during a new dispatch chain?